test-requirements.txt

```diff
@@ -0,0 +1,9 @@
+# TMP dependency to include https://github.com/akretion/odoo-test-helper/pull/1
+git+git://github.com/simahawk/odoo-test-helper.git@master#egg=odoo-test-helper
```

Comment:
@sebastienbeau can you help on this? :)

Reply:
I can work on it tomorrow  PM

Reply:
@sebastienbeau any update? This is blocking all other PRs.

Reply:
should be fixed w/ last commit